File: client/src/interfaces/User.dto.ts
export interface UserDto {
  fullname: string;
  username: string;
  email: string;
  avatarUrl?: string;
  password?: string;
  location?: string;
  newPassword?: string;
  repeatNewPassword?: string;

  phone?: string;
  role?: string;
  //statusId?: string;
  //id?: string;

  //show?: boolean;
  //errorMesage?: boolean;
}

File: server/goadventures/src/main/java/io/softserve/goadventures/dto/UserDto.java
package io.softserve.goadventures.dto;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
public class UserDto {
    private String fullname;
    private String username;
    private String email;
    private String location;
    private String avatarUrl;
    private int phone;
}
